{
  "_annotation": {
    "cve_id": "CVE-2023-21876",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2023/CVE-2023-21876.json",
    "reason": "Improve version ranges to indicate fix"
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:oracle:mysql:*:*:*:*:*:*:*:*",
                "matchCriteriaId": "D49A5DEB-E08E-5D7A-ADE3-45DB786D5C0C",
                "versionEndExcluding": "8.0.32",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          },
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:oracle:mysql_server:*:*:*:*:*:*:*:*",
                "matchCriteriaId": "16DD179E-B4A0-53B7-AF6A-0248F40963AE",
                "versionEndExcluding": "8.0.32",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}